[2022-02-28 14:59:11,227] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-02-28 14:59:11,236] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-02-28 14:59:11,236] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 14:59:11,236] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-02-28 14:59:11,236] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 14:59:11,247] {taskinstance.py:1264} INFO - Executing <Task(BranchPythonOperator): new_comic_available> on 2022-01-13 00:00:00+00:00
[2022-02-28 14:59:11,249] {standard_task_runner.py:52} INFO - Started process 34426 to run task
[2022-02-28 14:59:11,255] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'comic_app_v2', 'new_comic_available', 'scheduled__2022-01-13T00:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/comic_app_v2.py', '--cfg-path', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmp05tm5n7q', '--error-file', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmpfzuh1848']
[2022-02-28 14:59:11,257] {standard_task_runner.py:77} INFO - Job 137: Subtask new_comic_available
[2022-02-28 14:59:11,305] {logging_mixin.py:109} INFO - Running <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-02-28 14:59:11,345] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Adam
AIRFLOW_CTX_DAG_ID=comic_app_v2
AIRFLOW_CTX_TASK_ID=new_comic_available
AIRFLOW_CTX_EXECUTION_DATE=2022-01-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-13T00:00:00+00:00
[2022-02-28 14:59:11,350] {logging_mixin.py:109} INFO - 跟紀錄比較，有沒有新連載？
[2022-02-28 14:59:11,350] {python.py:175} INFO - Done. Returned value was: yes_generate_notification
[2022-02-28 14:59:11,351] {skipmixin.py:140} INFO - Following branch yes_generate_notification
[2022-02-28 14:59:11,351] {skipmixin.py:171} INFO - Skipping tasks ['no_do_nothing']
[2022-02-28 14:59:11,374] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=comic_app_v2, task_id=new_comic_available, execution_date=20220113T000000, start_date=20220228T065911, end_date=20220228T065911
[2022-02-28 14:59:11,397] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-28 14:59:11,429] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-28 15:05:36,301] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-02-28 15:05:36,309] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-02-28 15:05:36,309] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 15:05:36,309] {taskinstance.py:1244} INFO - Starting attempt 1 of 3
[2022-02-28 15:05:36,310] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-02-28 15:05:36,319] {taskinstance.py:1264} INFO - Executing <Task(BranchPythonOperator): new_comic_available> on 2022-01-13 00:00:00+00:00
[2022-02-28 15:05:36,322] {standard_task_runner.py:52} INFO - Started process 34998 to run task
[2022-02-28 15:05:36,327] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'comic_app_v2', 'new_comic_available', 'scheduled__2022-01-13T00:00:00+00:00', '--job-id', '211', '--raw', '--subdir', 'DAGS_FOLDER/comic_app_v2.py', '--cfg-path', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmpffr1e4gw', '--error-file', '/var/folders/jb/grqrlwkd311flwk_zt30r7wr0000gn/T/tmpk06k9hs1']
[2022-02-28 15:05:36,329] {standard_task_runner.py:77} INFO - Job 211: Subtask new_comic_available
[2022-02-28 15:05:36,377] {logging_mixin.py:109} INFO - Running <TaskInstance: comic_app_v2.new_comic_available scheduled__2022-01-13T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-02-28 15:05:36,417] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Adam
AIRFLOW_CTX_DAG_ID=comic_app_v2
AIRFLOW_CTX_TASK_ID=new_comic_available
AIRFLOW_CTX_EXECUTION_DATE=2022-01-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-13T00:00:00+00:00
[2022-02-28 15:05:36,421] {logging_mixin.py:109} INFO - 跟紀錄比較，有沒有新連載？
[2022-02-28 15:05:36,421] {python.py:175} INFO - Done. Returned value was: no_do_nothing
[2022-02-28 15:05:36,422] {skipmixin.py:140} INFO - Following branch no_do_nothing
[2022-02-28 15:05:36,422] {skipmixin.py:171} INFO - Skipping tasks ['yes_generate_notification']
[2022-02-28 15:05:36,444] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=comic_app_v2, task_id=new_comic_available, execution_date=20220113T000000, start_date=20220228T070536, end_date=20220228T070536
[2022-02-28 15:05:36,470] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-28 15:05:36,506] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
